mm2src/coins/nft/storage/wasm/nft_idb.rs

```diff
@@ -28,5 +44,8 @@ impl DbInstance for NftCacheIDB {
 }
 
 impl NftCacheIDB {
+    /// Get a reference to the underlying `IndexedDb` instance.
+    ///
+    /// This method allows for direct interaction with the raw database, bypassing any abstractions.
     pub(crate) fn get_inner(&self) -> &IndexedDb { &self.inner }
```

Comment:
@laruh I think we should consider making `get_inner` part of `DbInstance trait member` as it's used everywhere we've created IndexedDB cc. @onur-ozkan

Reply:
I dont think that all implementations need it

![image](https://github.com/KomodoPlatform/komodo-defi-framework/assets/52405288/07938c73-7c65-4296-bf4e-aa5a7c95c617)
note: ignore mm_err
![image](https://github.com/KomodoPlatform/komodo-defi-framework/assets/52405288/8bb78698-2711-4828-bedd-35f8776a2450)

Reply:
oh ok thanks